docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc

```diff
@@ -0,0 +1,191 @@
+[role="xpack"]
+[[ml-lang-ident]]
+=== {lang-ident-cap}
+
+experimental[]
+
+{lang-ident-cap} `model_id`: `lang_ident_model_1`
+
+{lang-ident-cap} is an {infer} trained model that you can use to determine in 
```

Comment:
If the model ID is required, perhaps it can be moved here?:
```suggestion
{lang-ident-cap} is an {infer} trained model (`lang_ident_model_1`) that you can use to determine the language of text.
```